src/test/java/org/broadinstitute/hellbender/utils/clipping/ReadClipperUnitTest.java

```diff
@@ -387,4 +392,60 @@ public void testRevertEntirelySoftclippedReads() {
         Assert.assertEquals(clippedRead.getStart(), getSoftStart(read));
     }
 
+
+    @Test
+    public void testSoftClipBothEndsByReferenceCoordinates() {
+        for (Cigar cigar : cigarList) {
+            GATKRead read = ReadClipperTestUtils.makeReadFromCigar(cigar);
+            int alnStart = read.getStart();
+            int alnEnd = read.getEnd();
+            int readLength = alnStart - alnEnd;
+            for (int i = 0; i < readLength / 2; i++) {
+                GATKRead clippedRead = ReadClipper.softClipBothEndsByReferenceCoordinates(read, alnStart + i, alnEnd - i);
+                Assert.assertTrue(clippedRead.getStart() >= alnStart + i, String.format("Clipped alignment start is less than original read (minus %d): %s -> %s", i, read.getCigar().toString(), clippedRead.getCigar().toString()));
+                Assert.assertTrue(clippedRead.getEnd() <= alnEnd + i, String.format("Clipped alignment end is greater than original read (minus %d): %s -> %s", i, read.getCigar().toString(), clippedRead.getCigar().toString()));
+                assertUnclippedLimits(read, clippedRead);
+            }
+        }
+    }
+
+    // This test depends on issue ##### as it tests the current behavior of the clipping operation
+    @Test
+    public void testSoftClippingOpEdgeCase() {
+        final SAMFileHeader header = new SAMFileHeader();
+        header.setSequenceDictionary(hg19GenomeLocParser.getSequenceDictionary());
+
+        GATKRead read = ReadClipperTestUtils.makeReadFromCigar("8M");
+        ReadClipper clipper = new ReadClipper(read);
+        ClippingOp op = new ClippingOp(0, 7);
+        clipper.addOp(op);
+        GATKRead softResult = clipper.clipRead(ClippingRepresentation.SOFTCLIP_BASES);
+        Assert.assertEquals(softResult.getCigar().toString(), "7S1M");
+    }
+
+
+//
+//    Test pending resolution of issue #
+//    @Test
```

Comment:
Instead of commenting this test out, can you switch it to `@Test(enabled = false)`